test/unit/lexer/token/equality.d

```diff
@@ -0,0 +1,349 @@
+module lexer.token.equality;
+
+import dmd.frontend : deinitializeDMD;
+import dmd.globals : Loc;
+import dmd.tokens : TOK;
+
+import support : afterEach;
+
+@afterEach deinitializeFrontend()
+{
+    deinitializeDMD();
```

Comment:
I don't really get all these functions that just call other functions. Phobos is full of them, and it's not for the better.

Reply:
`deinitializeFrontend` is called automatically by the unit test runner after each `unittest` block. `deinitializeDMD` will deinitialize the frontend.